cdap-ui/app/cdap/components/EntityListView/index.js

```diff
@@ -397,7 +397,7 @@ class EntityListView extends Component {
     };
 
     if (typeof query === 'string' && query.length) {
-      params.query = `${query}*`;
+      params.query = query.charAt(query.length - 1) === '*' ? query : `${query}*`;
```

Comment:
what happens if a user enters ***? Should we even handle it? I don't think so - that's a user error.